packages/ripple-binary-codec/src/binary.ts

```diff
@@ -66,8 +66,9 @@ function serializeObject(object: JsonObject, opts: OptionObject = {}): Buffer {
   const filter = signingFieldsOnly
     ? (f: FieldInstance): boolean => f.isSigningField
     : undefined
-
-  coreTypes.STObject.from(object, filter).toBytesSink(bytesList)
+  ;(coreTypes.STObject as typeof STObject)
```

Comment:
Why do all the types need to be casted now?

Reply:
Before I believe they were `any` - This specific instance looks at a single type which is in `coreTypes` which we know more specifically than the BaseClass. (So we cast it down to the level we need for the more specific `from` function)